lang/qore-jni-module/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem          1.0
 
 PortGroup           github 1.0
 github.setup        qorelanguage module-jni 2.0.6 v
+revision            1
```

Comment:
I don't think that removing non-default variants warrants increasing the `revision`

Reply:
Ok, I removed the revision bump.